code/modules/projectiles/pins.dm

```diff
@@ -28,7 +28,7 @@
 			var/obj/item/gun/G = target
 			var/obj/item/firing_pin/old_pin = G.pin
 			if(old_pin && (force_replace || old_pin.pin_removeable))
-				to_chat(user, span_notice("You remove [old_pin] from [G]."))
+				balloon_alert(user, "firing pin removed")
```

Comment:
if you like brownie points get rid of the single-letter var here, but it's pretty optional